docker-compose-dev.yml

```diff
@@ -1,9 +1,70 @@
 version: "3"
```

Comment:
Please rename this file to `docker-compose.dev.yml`.

Reply:
done